src/effects/lv2/LV2Instance.cpp

```diff
@@ -101,7 +101,8 @@ size_t LV2Instance::ProcessBlock(EffectSettings &,
    const float *const *inbuf, float *const *outbuf, size_t size)
 {
    using namespace LV2Symbols;
-   assert(size <= GetBlockSize());
+   if (size <= GetBlockSize())
```

Comment:
I do not yet understand why assertion was rewritten this way and I cannot debug this code due to the crash earlier

Reply:
I have checked - this rewrite now skips destructive processing altogether

Reply:
Oh stupid I should negate the condition